app/editor/src/App.tsx

```diff
@@ -3,7 +3,7 @@ import 'react-tooltip/dist/react-tooltip.css';
 
 import { ReactKeycloakProvider } from '@react-keycloak/web';
 import { LayoutAnonymous } from 'components/layout';
-import { UploadContextWrapper } from 'components/upload';
+import { UploadContextWrapper } from 'features/content';
```

Comment:
Upload is tightly coupled with the content form.  At some point it would be nice to make it generic.  Until then it belongs close to the content form.